okio-behavior/src/main/java/okio/test/ByteRate.java

```diff
@@ -0,0 +1,59 @@
+package okio.test;
+
+import java.util.concurrent.TimeUnit;
+
+/** A byte count expressed over a period of time for determining transfer rates. */
+public final class ByteRate {
```

Comment:
should this have helper functions ?

```
ByteRate byteRate = new ByteRate();
byteRate.kBps(28);
byteRate.mBps(28);
```

The bytes/bits nonsense burns us a bit here here.